pi-blaster.c

```diff
@@ -32,7 +32,7 @@
 
 #define _POSIX_C_SOURCE 200809L
 #define _XOPEN_SOURCE   700
-#define _BSD_SOURCE
+#define _DEFAULT_SOURCE
```

Comment:
Can you explain this change?

Reply:
There where warning while compiling. This settings fixed it.

Reply:
For future readers, this stack overflow answer gives a bit more details: https://stackoverflow.com/questions/29201515/what-does-d-default-source-do